src/app/newSettings/PreferencesDialog.tsx

```diff
@@ -53,50 +52,53 @@ const PreferencesDialog = (props: PreferencesDialogProps) => {
     setActiveSection(selectedNavSection);
   };
 
-  const onClose = () => {
+  const onClosePreferences = () => {
     dispatch(uiActions.setIsPreferencesDialogOpen(false));
     navigationService.closePreferencesDialog();
   };
 
   return (
     <Modal
       maxWidth="max-w-4xl"
-      className="dark:dbg-surface m-0 flex h-640 shadow-sm"
+      className="m-0 flex h-640 overflow-hidden shadow-sm"
       isOpen={isPreferencesDialogOpen}
-      onClose={() => onClose()}
+      onClose={() => onClosePreferences()}
+      // preventClosing={isToastNotificationOpen}
```

Comment:
Does this have to work or do we remove the line?

Reply:
Removed